### PR TITLE
Vijay content workflow

### PR DESCRIPTION
**Pull request by vijay-deriv**
workflow change to deploy a latest production tag when submitted in cms system

### CODE REVIEW

**Comment by vijay-deriv** [comment]
made a PR please review
cc : @shudarshon-binary

File: .github/workflows/content.yml [hunk 1]
```diff
@@ -44,6 +44,20 @@ jobs:
         steps:
             - name: Checkout 🛎️
               uses: actions/checkout@v2.3.1
+              
+            - name: fetch git tags
+              id: get_tag_name
+              run: |
+                   git fetch --prune --unshallow --tags
+                   GIT_TAG_NAME=`git tag --format='%(creatordate:short)%09%(refname:strip=2)' --sort=creatordate | grep production | awk '{ print $2; }' | tail -1`
```
Comment (shudarshon-binary) [inline .github/workflows/content.yml:52]: lets not use `creatordate` for sorting instead use `taggerdate`. can we please reproduce fetching git tag by using this `git for-each-ref --sort=taggerdate --format '%(tag)' | grep production | tail -1` ?

Comment (shudarshon-binary) [reply .github/workflows/content.yml:52]: `tail -10` is basically reducing grepping over along list of tags.

Comment (shudarshon-binary) [reply .github/workflows/content.yml:52]: LGTM

**Review by shudarshon-binary — APPROVED** [review]

**Pull request merged**